lib/brook/event.ex

```
@@ -68,15 +68,15 @@ defmodule Brook.Event do
       {:ok, serialized_event} ->
         :ok = apply(driver.module, :send_event, [instance, type, serialized_event])
 
-      {:error, reason} ->
+      {:error, reason} = error ->
         Logger.error(
           "Unable to send event: type(#{type}), author(#{author}), event(#{inspect(event)}), error reason: #{
             inspect(reason)
           }"
         )
-    end
 
-    :ok
+        error
```

Was the :ok an intentional choice to try to always return ok even if we erred (and the spec was incorrect) or did we just miss returning the error? Just wanted to double check.

---

The spec says it could return `{:error, Brook.reason()}`. This is the only feasible place to do so. If we'd rather change the spec, I can do that. But we match against that error tuple elsewhere.

---

Ok, if we match elsewhere against it returning an error then that makes sense. I just thought we might be intentionally swallowing the error for some reason and the spec was incorrect. Thanks.